test/rules/named-return-values.ts

```diff
@@ -0,0 +1,62 @@
+import assert from 'assert';
+import linter from '../index';
+const contractWith = require('solhint/test/common/contract-builder').contractWith;
+
+describe('Linter - named-return-values', () => {
+  it('should raise error for unnamed return values', () => {
+    const code = contractWith('function test() returns (uint) {}');
+    const report = linter.processStr(code, {
```

Comment:
save a const with the rule object outside of the test cases and reuse it in every single one, making the `.processStr` a one-line thing;

```
const config = {
      rules: { 'named-return-values': 'error' },
    };
```

Same for every test file